spec/spec_helper.rb

```diff
@@ -6,6 +6,8 @@
 
 Dir['./spec/support/**/*.rb'].each { |file| require file }
 
+ENV['TESTING'] = '1'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.